components/posts/post-item.js

```diff
@@ -37,13 +34,9 @@ function PostItem(props) {
               className='object-cover'
             />
           </div>
-          <div className="bg-gray-800 p-4 h-52 flex flex-col items-center"
+          <div className="bg-gray p-4 h-52 flex flex-col items-center"
           >
-            <h3 className="text-white text-2xl text-extrabold mb-5"
-            style={{
-              fontFamily: 'Oswald',
-              fontSize: '2rem'
-              }}
+            <h3 className="text-white text-4xl font-oswald text-extrabold mb-5"
```

Comment:
This is really cool. It's just rad to see how immediate the code gets "leaner" as soon as you imported the font and used the 'font-oswald' property.